packages/ringcentral-widgets-test/test/integration-test/CallCtrlPage/helper.js

```diff
@@ -17,7 +17,9 @@ import { timeout } from '../shared';
 
 export async function makeOutboundCall(phone) {
   mock.device(deviceBody);
-  phone.webphone.sessions.forEach(session => phone.webphone.hold(session.id));
+  phone.webphone.sessions.forEach(async (session) => {
```

Comment:
```javascript
(async function(){
  [100,200,300].forEach(async function(i){
    await new Promise((resolve)=>{
      setTimeout(resolve, i)
    })
    console.log(i)
  });
  console.log(400)
})()
// The result would be like:
// 400
// Promise {<resolved>: undefined}
// 100
// 200
// 300
```
so what's the meaning of it

Reply:
so it should be like:

const funcs = [100, 200, 300].map(count => new Promise(resolve => setTimeout(resolve, count)));

(async function() {
  await Promise.all(funcs);
})()

right?

Reply:
yes, or you can:
```
for(let time of [100, 200, 300]){
  await new Promise(resolve => setTimeout(resolve, time))
}
// other codes
```